tripal_chado/includes/TripalFields/data__sequence_record/data__sequence_record.inc

```diff
@@ -127,10 +127,20 @@ class data__sequence_record extends ChadoField {
     ];
     $seqs = chado_get_feature_sequences(['feature_id' => $feature->feature_id], $options);
     $featurelocs = [];
+
     foreach ($seqs as $seq) {
       $featureloc = $this->getFeatureLoc($seq['featureloc_id']);
       $coords = $this->getSequenceCoords($featureloc);
 
+      # SOFIA: if the feature is a srcfeature don't derive sequence from itself
+      if ($feature->feature_id !=  $featureloc->srcfeature_id){
+        # i am not quite sure what the $option is doing, i am going to set it to 0 since there is no seq derived from parent
+        $options = [
+          'derive_from_parent' => 0,
+         ];
+         continue;
+      }
+
```

Comment:
I'm good with this. Can you fix the comment style so #'s are two forwards slashes?  (Drupal coding style)

Reply:
Done.